examples/pylab_examples/multipage_pdf.py

```diff
@@ -18,7 +18,8 @@
     pdf.savefig()  # saves the current figure into a pdf page
     plt.close()
 
-    plt.rc('text', usetex=True)
+    plt.rc('text', usetex=True)  # if LaTeX is not installed or error caught,
+                                  # change to `usetex=False` to make it work
```

Comment:
This line is not aligned any more; however, I think it might be simpler to just place the comment above the code (if it fits within 80 characters).